packages/ui/src/composites/wui-tabs/index.ts

```diff
@@ -13,36 +14,90 @@ export class WuiTabs extends LitElement {
 
   @property() public onTabChange: (index: number) => void = () => null
 
+  @property({ type: Array }) public buttons: HTMLButtonElement[] = []
+
   @state() public activeTab = 0
 
+  @state() public localTabWidth = '100px'
+
+  @state() public isDense = false
+
   // -- Render -------------------------------------------- //
   public override render() {
-    const isDense = this.tabs.length > 3
+    this.isDense = this.tabs.length > 3
 
     this.style.cssText = `
       --local-tab: ${this.activeTab};
-      --local-tab-width: ${isDense ? '80px' : '100px'}
+      --local-tab-width: ${this.localTabWidth};
+      --local-dense-tab-width: max-content;
```

Comment:
`--local-dense-tab-width: max-content;` does this need to be here, seems like it is static so we can just include in css?